src/Microsoft.AspNet.Mvc.Core/OptionDescriptors/ModelBinderDescriptorExtensions.cs

```diff
@@ -80,5 +80,24 @@ public static class ModelBinderDescriptorExtensions
             descriptors.Insert(index, descriptor);
             return descriptor;
         }
+
+
+        /// <summary>
+        /// Removes all instances of <typeparamref name="TInstance"/> from a list of ModelBinderDescriptors.
+        /// </summary>
+        /// <typeparam name="TInstance">A type that represents an <see cref="IModelBinder"/>.</typeparam>
+        /// <param name="descriptors">A list of ModelBinderDescriptors.</param>
+        public static void RemoveTypesOf<TInstance>(
+            [NotNull] this IList<ModelBinderDescriptor> descriptors)
+            where TInstance : class, IModelBinder
+        {
+            for (int i = descriptors.Count - 1; i >= 0; i--)
+            {
+                if (descriptors[i].OptionType == typeof(TInstance))
```

Comment:
@yishaigalatzer Let me know if you think I should add `<remarks/>` and what it should say.